Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyFragment.kt

```diff
@@ -62,79 +73,151 @@ class SurveyFragment : Fragment() {
                 ViewGroup.LayoutParams.MATCH_PARENT,
                 ViewGroup.LayoutParams.MATCH_PARENT
             )
+
             setContent {
                 JetsurveyTheme {
-                    val state = viewModel.uiState.observeAsState().value ?: return@JetsurveyTheme
-                    AnimatedContent(
-                        targetState = state,
-                        transitionSpec = {
-                            fadeIn() + slideIntoContainer(
-                                towards = AnimatedContentScope
-                                    .SlideDirection.Up,
-                                animationSpec = tween(ANIMATION_SLIDE_IN_DURATION)
-                            ) with
-                                fadeOut(animationSpec = tween(ANIMATION_FADE_OUT_DURATION))
+                    val isSurveyComplete by viewModel.isSurveyComplete.observeAsState(false)
+
+                    if (isSurveyComplete) {
+                        // Note, results are hardcoded for this demo; in a complete app, you'd
+                        // likely send the survey responses to a backend to determine the
+                        // result and then pass them here.
+                        SurveyResultScreen(
+                            title = stringResource(R.string.survey_result_title),
+                            subtitle = stringResource(R.string.survey_result_subtitle),
+                            description = stringResource(R.string.survey_result_description),
```

Comment:
What is the reasoning behind passing these string resources and making them dynamic? They're not backed by any data, and this is the only calling spot. For simplicity, I'd consider just hard-coding them in `SurveyResultScreen`.

Reply:
I was originally going to have them come from the ViewModel and pass them through the Composables, but it didn't really make sense given they were still hardcoded. I like your idea of hardcoding them in SurveyResultScreen instead, so I've done that.